### PR TITLE
assures shipping method is set on firecheckout order creation call

### PR DESCRIPTION
**Pull request by mccottry**
https://app.asana.com/0/544708310157130/770550762170012

### CODE REVIEW

File: app/code/community/Bolt/Boltpay/controllers/OrderController.php [hunk 1]
```diff
@@ -170,6 +168,9 @@ public function firecheckoutcreateAction()
                 }
             }
 
+            $checkout->applyShippingMethod($this->getRequest()->getPost('shipping_method', false));
+            $quote->getShippingAddress()->setShippingMethod($this->getRequest()->getPost('shipping_method', false));
```
Comment (YevhenShBolt) [inline app/code/community/Bolt/Boltpay/controllers/OrderController.php]: Does exist a difference between these two methods?

`$checkout->applyShippingMethod(...)`
and
`$quote->getShippingAddress()->setShippingMethod(...)`

It seems that they should apply the same thing, but I do not have the Firecheckout module right now.

Comment (mccottry) [reply app/code/community/Bolt/Boltpay/controllers/OrderController.php]: @YevhenShBolt 
Yes.  There is a difference.

Comment (YevhenShBolt) [reply app/code/community/Bolt/Boltpay/controllers/OrderController.php]: It seemed to me that these both methods add the same data to the same quote.

Comment (mccottry) [reply app/code/community/Bolt/Boltpay/controllers/OrderController.php]: @YevhenShBolt I've made an addition pull request to address this issue.

@daisy1754 Please note that https://github.com/BoltApp/bolt-magento1/pull/151, once approved, should be merged to this PR first before this is merged to develop.

**Pull request merged**